public/app/features/dashboard/components/PanelEditor/OptionsPaneContent.tsx

```diff
@@ -35,12 +36,13 @@ export const OptionsPaneContent: React.FC<Props> = ({
   const styles = getStyles(theme);
   const [activeTab, setActiveTab] = useState('options');
   const [isSearching, setSearchMode] = useState(false);
+  const currentData = usePanelLatestData(panel);
```

Comment:
this component still takes data passed from PanelEditor (that comes from redux). Maybe we can remove the redux panel data and use a similar hook for the transformations tab?

Reply:
But transformations tab needs the original data and the transformations are applied "inline" in the TransformationsEditor.

Reply:
the transformations are not in the OptionsPaneContent tab. Just saying we should remove the data property on the OptionsPaneContent component and currentData to PanelOptionsTab as well. 

Then I think we should use usePanelLatestData in TransformationsTab , that we can remove the querySubscription and getData from PanelEditor redux store. all we need to do is add a (transform: bool) prop to usePanelLatestData so we can disable transforms then we can use it for TransformationsTab

Reply:
ah yeah, i see now

Reply:
> Then I think we should use usePanelLatestData in TransformationsTab , that we can remove the querySubscription and getData from PanelEditor redux store. all we need to do is add a (transform: bool) prop to usePanelLatestData so we can disable transforms then we can use it for TransformationsTab 

gona leave this for now and add as followup as this requries some refactor of TransformationsEditor